index.html

```diff
@@ -12,53 +12,60 @@
-            </div>
+    <main class="main-container">        
+        <header>          
+          <div class="toolbar">
+            <a href=""><h1>My Logo</h1></a>
+            
+            <div class="menu-desktop">
+              <a href="#" class="menu-opt">Portfolio</a>
+              <a href="#" class="menu-opt">About</a>
+              <a href="#" class="menu.opt">Contact</a>
+            </div>        
+
+            <nav class="mobile-nav"> 
+              <img src="./images/hamburger-icon.svg" alt="Menu">
+            </nav>
+          </div>
         </header>
 
-        <section class="headline">
-            <h1><br>I'm Carlos<br>Glad to see<br>you!</h1> 
-            <p>I'm a software developer! I can help you<br>
-              build a product , feature or website Look<br>
-              through some of my work and<br>
-              experience! If you like what you see and<br>
-              have a project you need coded, don’t<br>
-              hestiate to contact me.
+        <section class="section-headline">
+          <div class="headline">
+            <h1>I'm Carlos<br>
+                Glad to see you!</h1> 
+            <p>
+              I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
             </p>
             <h3>LET'S CONNECT</h3>
-              <div >
-                <ul class="icons-social" >
-                  <li>
-                    <img src="images/github.png" class="social" alt="Github">
-                  </li>
-                  <li>
-                    <img src="images/Linkedin.png" class="social" alt="Linkedin">
-                  </li>                               
-                  <li>
-                    <img src="images/angellist.png" class="social" alt="angellist">
-                  </li>
-                  <li>
-                    <img src="images/twitter.png" class="social" alt="twitter">
-                  </li>
-                  <li>
-                    <img src="images/medium.png" class="social" alt="Medium">
-                  </li>
-                </ul>
-              </div>
+            <div>
+              <ul class="icons-social" >
+                <li>
+                  <img src="images/github.png" class="social" alt="Github">
+                </li>
+                <li>
+                  <img src="images/Linkedin.png" class="social" alt="Linkedin">
+                </li>                               
+                <li>
+                  <img src="images/angellist.png" class="social" alt="angellist">
+                </li>
+                <li>
+                  <img src="images/twitter.png" class="social" alt="twitter">
+                </li>
+                <li>
+                  <img src="images/medium.png" class="social" alt="Medium">
+                </li>
+              </ul>
```

Comment:
- [OPTIONAL] Great job using `ul` and `li` tags 💯, However. I suggest using `<a>` tag before the `<img>` tag. Because they are social Links, (should be clickable) 👌